tinydb/database.py

```diff
@@ -77,6 +77,10 @@ def __init__(self, storage, table_name):
         self._storage = storage
         self._table_name = table_name
 
+    def construct_doc(self, key, val):
```

Comment:
Could we rename this to an internal method and also use something like `def _new_document(…)`?

Reply:
a new commit for that.

Reply:
Awesome! Now you just need to adapt the example in https://github.com/msiemens/tinydb/blob/a454998e65e43f7af44e008129d08a44fcfed1e6/tests/test_tinydb.py#L617-L618 🙂

Reply:
😂